From 63f8bfdc113a88bcac194498306a6665f95b8f54 Mon Sep 17 00:00:00 2001
From: Steve Kondik <steve@cyngn.com>
Date: Fri, 2 Sep 2016 01:25:47 -0700
Subject: [PATCH 228/296] Telephony: Add getDefaultDataSubId for compatibility

 * Some applications are still using getDefaultDataSubId, which
   has changed to the more verbose getDefaultDataSubscriptionId.
---
 telephony/java/android/telephony/SubscriptionManager.java | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/telephony/java/android/telephony/SubscriptionManager.java b/telephony/java/android/telephony/SubscriptionManager.java
index 88f4880a98f..b2e95787765 100644
--- a/telephony/java/android/telephony/SubscriptionManager.java
+++ b/telephony/java/android/telephony/SubscriptionManager.java
@@ -1234,6 +1234,11 @@ public class SubscriptionManager {
         }
     }
 
+    /** @hide */
+    public static int getDefaultDataSubId() {
+        return getDefaultDataSubscriptionId();
+    }
+
     /**
      * Return the SubscriptionInfo for default data subscription.
      *
-- 
2.11.0

